backend/fms_core/template_importer/importers/container_creation.py

```diff
@@ -14,12 +16,12 @@ def import_template_inner(self):
         for row_id, row_data in enumerate(containers_sheet.rows):
             container = {
                 'kind': row_data['Container Kind'],
```

Comment:
Do we wanna `str_cast_and_normalize` Container Kind?

Reply:
If this is a string we should i think. This is unlikely someone will input a number in there but better safe than sorry.

Reply:
```suggestion
'kind': str_cast_and_normalize(row_data['Container Kind']),
```

Reply:
```suggestion
                'kind': str_cast_and_normalize (row_data['Container Kind']),
```

Reply:
Oh right ... i should apply that !